index.js

```diff
@@ -2648,6 +2650,15 @@ RemoteObjectTemplate.bindDecorators = function (objectTemplate) {
             objectTemplate._setupProperty(targetKey, props, undefined, defineProperties);
             Object.defineProperties(target, defineProperties);
         };
+
+        function applyRuleSet(prop, ruleSet) {
+            if (prop instanceof Array && prop.length > 0 && ruleSet instanceof Array && ruleSet.length > 0) {
+                return prop.some(r=> ruleSet.indexOf(r) >= 0);
+            }
+            else {
+                return prop;
```

Comment:
if prop is empty array it will return an empty array - is this ok? i would expect a `false` in that scenario

Reply:
@hackerrdave - good catch, fixed the issue. if we use array for these attributes, then the application name must be listed.